src/features/game/types/achievements.ts

```diff
@@ -254,7 +254,7 @@ export const ACHIEVEMENTS: () => Record<AchievementName, Achievement> = () => ({
   "Bakers Dozen": {
     description: translate("bakersDozen.description"),
     progress: (gameState: GameState) => {
-      const cakeEvents = getKeys(CAKES()).map(
+      const cakeEvents = getKeys(COOKABLE_CAKES).map(
```

Comment:
```suggestion
      const cakeEvents = getKeys(COOKABLE_CAKES()).map(
```
You may need to add Bracket parentheses to this variable

Reply:
> You may need to add Bracket parentheses to this variable

Thanks you @eliasSFL for your answer.

This is precisely one of the differences with the previous value `CAKES`, which was a function. Here, `COOKABLE_CAKES` is a `Record<CookableName, Cookable>`. As a result, the () operator is not defined.

So I took inspiration from the solution used for the `Kiss the Cook` achievement, which uses the COOKABLES variable (which is also a `Record<CookableName,Cookable>`).

Reply:
Oh I see. Thanks for letting me know, I'm still learning as I go even though I've been contributing code for a while now.